packages/Confirmation/src/components/Content/Content.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+export default function Content() {
```

Comment:
A competing convention is to distinctly name this kind of component a `PropsCollector`.  

https://github.com/acl-services/paprika/blob/master/packages/Checkbox/src/CheckboxInputPropsCollector.js

Looks like you've followed the pattern in the `<OverflowMenu>` as closely as possible, but if you ignore that factor, which pattern do you like better?